scripts/vellum-to-hq

```diff
@@ -108,7 +108,7 @@ def build_vellum(path, branch, test=True):
 
 def get_old_vellum_rev(git):
     commit = str(git('log', '--oneline', '--grep', 'vellum:', '-n', '1'))
-    return re.search('[0-9a-f]+ vellum:([0-9a-f]+)', commit).group(1)
+    return re.search('[0-9a-f]+ (?:Revert ")?vellum:([0-9a-f]+)', commit).group(1)
```

Comment:
This will get the commit hash of the reverted version, but we really want the version before that, right? This is hard for me to reason about without an example. Are we talking about a sequence like this:

commits:
* `vellum:a` <- we want this (a)
* `vellum:b`
* `Revert vellum:b` <- we get this (b)